examples/sesv2/emailidentity-domain.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: sesv2.aws.crossplane.io/v1alpha1
+kind: EmailIDentity
+metadata:
+  name: example-domain
+spec:
+  deletionPolicy: Delete
+  forProvider:
+    configurationSetName: example-configset-name
+    dkimSigningAttributes: #only during first creation
+      domainSigningPrivateKey: base64_encoded_pk
+      domainSigningSelector: subdomain #Without suffix ._domainkey.mydomain
+    mailFromAttributes:
+      behaviorOnMxFailure: USE_DEFAULT_VALUE #USE_DEFAULT_VALUE | REJECT_MESSAGE
+      mailFromDomain: subdomain.mydomain
+    emailIDentity: mydomain
```

Comment:
@AaronME I think this (`emailIDentity` -> `emailIdentity`) will be fixed with the new code gen version. So the question is if we should wait for that first, or just make sure we get both merged before we do a release. Either way, one of them will have to be rebased. I think I lean on getting the SDK/code gen PRs merged first. They are fairly large and harder to review, and if we do the new resource PRs first we might end up with more complex "re-reviews" and rebases on the SDK PRs.

As for the rest of the code, I have not reviewed it fully, I just looked over it to help @kelvinwijaya forward due to lack of reviews.

I think it's also a question of what quality and review we should have for *alpha* resources. It might be good to get them merged, and then be stricter for alpha->beta graduation, when we have someone who knows the product and can follow up with fixes in the future, and we don't have enough maintainer capacity to fully understand and test the specific service.